printer_mods/Skyllaer/Webcam_mount/README.md

```diff
@@ -0,0 +1,25 @@
+# Webcam mount with gopro swivel
+==============================
+
+I was looking for a small footprint webcam for usage in the V2.4
+I used this one (160 degree's version) : https://nl.aliexpress.com/item/32898413562.html
+
+Installation is pretty straight forward.
+Materials needed :
+* 1x m3 heat insert (V2.4 style)
+* 2x m2 T-nuts
```

Comment:
Shouldn't these be M3 t-nuts?

Reply:
Sorry, I must've missed your comment here, you can push your changes to the same branch that your PR came from and the changes will automatically appear as part of this PR.